web/src/main/java/com/bazaarvoice/emodb/web/EmoService.java

```diff
@@ -139,7 +140,7 @@ public static void main(String... args) throws Exception {
         Namespace result = parser.parseArgs(first3Args);
 
         // Remove config-ddl arg
-        new EmoService(result.getString("config-ddl"), new File(result.getString("emo-config")))
+        new EmoService(result.getString("config_ddl"), new File(result.getString("emo_config")))
```

Comment:
why was this changed?

Reply:
From the docs:
```
For optional argument actions, the value of dest is normally inferred from the option strings. ArgumentParser generates the value of dest by taking the first long option string and stripping away the initial -- string. Any internal - characters will be converted to _ characters to make sure the string is a valid attribute name```

If we do not change these to underscores, they will return null, which will lead to NPE's.